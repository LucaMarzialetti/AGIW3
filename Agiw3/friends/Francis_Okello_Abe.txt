--url_fb https://www.facebook.com/adelakun.isaac.5
--friends
VM3849:3 Omotoyosi Etoo Ogunnubi
VM3849:3 Yetunde Abolade
VM3849:3 Adedamola Salako
VM3849:3 Hollarwunmi Holluwachemisola
VM3849:3 Agboola Temitope
VM3849:3 Aiyedogbon Temitayo
VM3849:3 Azeez Abiodun Adebisi Adeniji
VM3849:3 Comedian Mc Asiko
VM3849:3 Adeyemi Omoniyi Samuel
VM3849:3 David Samson
VM3849:3 Ayodele Michael Ola
VM3849:3 Prince Fadare Moses Hardeyinka
VM3849:3 Rhomorkeh Betty
VM3849:3 Olujitan Elijah Haryourholluwar
VM3849:3 Florence Hapholaryan Tosin
VM3849:3 TDzeal Tolulope Soyemi
VM3849:3 Hormorlahdey Ibukunoluwa Sayrah
VM3849:3 Wale Johnson Tosin
VM3849:3 Adeyemi Bamidele Ayokunle
VM3849:3 Adelakun Stephen
VM3849:3 Agbeyangi Timothy Olufemi
VM3849:3 Funmilola Olayemi
VM3849:3 Abolade Olawale Olanrewaju
VM3849:3 Josiah Oruh
VM3849:3 Balogun Pascal
VM3849:3 Lahwhal Fowoshere
VM3849:3 Akinola Mercy Igho-Osokor
VM3849:3 Holluwadharmsel Akanke Eyinjuolodumare
VM3849:3 Ajafem Emmanuel Ajayi
VM3849:3 Frank Kayode Omoiran
VM3849:3 Samuel Ogungbenro
VM3849:3 Olujitan Sarah Olayeancah
VM3849:3 Eunice Sobade
VM3849:3 Comedian Mc Ayanfe
VM3849:3 Olori Bamidele Suprella
VM3849:3 Certified Bee
VM3849:3 Prince Segun Arowosafe
VM3849:3 Ajala Dorcas Olulola
VM3849:3 Julius Kenedy
VM3849:3 Esther Ilesanmi
VM3849:3 Princess Oluwaseunfunmi
VM3849:3 Michael Allen
VM3849:3 Azanor Emmanuel Tegha
VM3849:3 Ayooluwa Olufade
VM3849:3 Michael Adelakun
VM3849:3 Adeoye Gbenga Joseph
VM3849:3 Obaoye Abiodun Daniel
VM3849:3 Aiyedogbon Oluwathobby Mr-celebrity
VM3849:3 Prince Ade Ade-Jesu Festus
VM3849:3 Gideon Balogun
VM3849:3 Oyebade Hardeywharley
VM3849:3 Israel Ige Adeniyi
VM3849:3 Fagbule Wunmi Maria
VM3849:3 Wale Adesola
VM3849:3 Shonubi Oluseun Titus
VM3849:3 Ojo Ebenezer
VM3849:3 Adeniji Philips
VM3849:3 George Praise
VM3849:3 Sammy Da Jazz Drummer
VM3849:3 Solomon Rachael
VM3849:3 Abiodun Rihana Akamo
VM3849:3 Pst S O Oke
VM3849:3 Gideon Olutola
VM3849:3 Adebowale Adenigbagbe
VM3849:3 Mc Kewentino
VM3849:3 Hizexcellence Sheckem Osorire
VM3849:3 Oluwatobiloba Olatunbosun Olasunkanmi
VM3849:3 Iyabo Daniel
VM3849:3 Ife Adelakun
VM3849:3 Holuwabunmi Janet
VM3849:3 Marie Holuwaphumelaryor Lharde
VM3849:3 Ilesanmi Kolawole
VM3849:3 Olusegun Gbadero
VM3849:3 Olujitan Dosas
VM3849:3 Taiwo Segun
VM3849:3 Shokunbi Sky D Oluwadamilola
VM3849:3 Idowu Olamide Akinsohun
VM3849:3 Kosemani Holuwasegun
VM3849:3 Fatai Olowookere Flowkee
VM3849:3 Oginni Gideon Odunayo
VM3849:3 Adedeji Kayode
VM3849:3 Anuoluwapo Olusanya
VM3849:3 Olu Fad MJ
VM3849:3 Emmanuel Jihingbólá Abiona
VM3849:3 Beatrice Romoke
VM3849:3 Doyin Deji
VM3849:3 Olubunmi Joel
VM3849:3 Oyeledun Oluwakayode
VM3849:3 Aderibigbe Opeyemi Otaobayomie
VM3849:3 Mary Olushola
VM3849:3 Glorious King Emmanuel
VM3849:3 Olubunmi Nathaniel
VM3849:3 Horlargeedey Horlorunphemmy Hormorlardey
VM3849:3 Aborisade Timothy Oluseye
VM3849:3 George-Akintola Phebean Olutosin
VM3849:3 Dammybarz Olubunmi
VM3849:3 Olujitan Emmanuel
VM3849:3 Yalumo Elijah
VM3849:3 Ebire Oluwaseyi Esther
VM3849:3 Samuel Adewale
VM3849:3 Elizabeth Olaoluwa Oyeyinka
VM3849:3 Emmanuel Abayomi Raphael
VM3849:3 Benedicta Ememameh
VM3849:3 Damilola Abolade
VM3849:3 Fakanlu Abidemi
VM3849:3 Layi Lore
VM3849:3 Agbolade Oluwasegun Kayode
VM3849:3 Harlarape Timmybankz
VM3849:3 Vatop Shegzy
VM3849:3 Isaac Adumati
VM3849:3 Olaniyan Oluwadamilola Debbie
VM3849:3 Henry John
VM3849:3 Akinbo Kehinde
VM3849:3 Titilayo Abolade
VM3849:3 Allen Gideon
VM3849:3 Olufade Busayo
VM3849:3 Kenny K Adekola
VM3849:3 Bayo Micheal Ajidara
VM3849:3 Victor Ohaeri
VM3849:3 Oni Mayowa
VM3849:3 Funmilayo Fadeyibi
VM3849:3 Owolabi Opeyemi Olaitan
VM3849:3 Oyebode Funmilayo Eunice
VM3849:3 Segun Temidayo
VM3849:3 Timsho Sholola
VM3849:3 Ayantunji Lanre Taofik
VM3849:3 Ife Adelakun
VM3849:3 Nonso Ojewe
VM3849:3 Ajayi Olamiji Idumah
VM3849:3 Akintola Ezekiel O
VM3849:3 Stephen Adelakun
VM3849:3 MC Kefty Comedian
VM3849:3 Hope Nnaa
VM3849:3 Hope Edebor
VM3849:3 Tolulope Omotayo Mewojuaye
VM3849:3 Janet Amoo Abolade
VM3849:3 Taiwo Ebenezer
VM3849:3 Deborah Fapohunda
VM3849:3 Ogunsanya Oluwaseyi Adebanjo
VM3849:3 Mike Ola Ayo
VM3849:3 Caleb Oluwatomilola Taiwo
VM3849:3 Rapheal Olanrewaju
VM3849:3 Godwin Akintola
VM3849:3 Cyril Uchechukwu
VM3849:3 Olaleye Olanike
VM3849:3 Daniela Blessing
VM3849:3 Lola Olasinde Oniyelu
VM3849:3 Oluwatosin Adeola Ajayi
VM3849:3 Ola Jacob Abolade
VM3849:3 Adeniyi Oluwafunmilayo
VM3849:3 Ademola Olubanwo
VM3849:3 Hasna Witnes
VM3849:3 Akinpelu Matthew Ifeoluwa
VM3849:3 Abe Shade
VM3849:3 Segun Timothy Adanks
VM3849:3 Adenipekun Ayo Adekunle
VM3849:3 Olaleye Tolulope
VM3849:3 Horlahlehyeh Dhamielawlah Sticool
VM3849:3 Gbemi Ayedogbon
VM3849:3 Oludiya Isaac Oluwafemi
VM3849:3 Samuel Adegboyega Uni
VM3849:3 Bamitale Orimidu
VM3849:3 Josiah Blinky
VM3849:3 Isaiah Oluwasegun
VM3849:3 Princess Ukaps
VM3849:3 Oluwafemi Olugbodi
VM3849:3 Esther Jumai Eleke
VM3849:3 Haryourmeday Aiyedogbon
VM3849:3 Oluwatobi Olubukola Akinbo
VM3849:3 Esther Dickson
VM3849:3 Fakan Ajoke
VM3849:3 James Anuoluwapo
VM3849:3 Rhoda Oluwatoyin Agbeja
VM3849:3 Seun Omotola
VM3849:3 Lanre F Sama Qca
VM3849:3 Jide Junior Page
VM3849:3 Fatoye Emmanuel Flexy
VM3849:3 Omohkhaphen Papaz Idumah
VM3849:3 Bamigbola Titilope
VM3849:3 Liasu Horppy
VM3849:3 Prince Shonaike Olakunle Isaiah
VM3849:3 Olufemi Taiwo
VM3849:3 Omnipotent Reigneth
VM3849:3 Adeoyo Moyosoluwa
VM3849:3 Jephthah Oruh
VM3849:3 Damilare Adeniyi
VM3849:3 Yung Kelvin
VM3849:3 Oladokun Funmilayo
VM3849:3 Sammiekeis Isaiah
VM3849:3 Soji Ibidun
VM3849:3 Alashela Bolakale Alabi
VM3849:3 Olas Jane
VM3849:3 Busayo Vicky Oluyori
VM3849:3 Adeolu Adeuga
VM3849:3 Abbey Matthew
VM3849:3 Aduragbemi Adekoya
VM3849:3 Opeoluwa Ipoade
VM3849:3 Godwin Danyel
VM3849:3 Aruwajoye Solomon Oluwasheun
VM3849:3 Victor Aworuwa
VM3849:3 Hermagesty Esther Abraham
VM3849:3 Nurry Keshtu Seid
VM3849:3 Happiness Timilehin Ojo
VM3849:3 Mhiz Oluwabukunmi Bidex
VM3849:3 Adenigbagbe Johnson
VM3849:3 Joseph Odunayo Hadeyemi
VM3849:3 Princess Rachael
VM3849:3 Haybell Oluwatosin Adekola
VM3849:3 Aiyedogbon Roseline Titilayo
VM3849:3 Olowomeye Lawrence
VM3849:3 Towakennu Emmanuel
VM3849:3 Jimson Samuel Olumide
VM3849:3 Adenigbagbe Johnson Joseph
VM3849:3 Fagbule Olushola Mercy
VM3849:3 Agbeyangi Olutobi Pastor
VM3849:3 Gbemisola Adebayo
VM3849:3 Monisola Abatan
VM3849:3 Olagunju Ebenezer Oluwaseun
VM3849:3 Tifase Ifeoluwa Olamilekan
VM3849:3 Tunde Fapohunda
VM3849:3 Helvie Jean Labiao Glinoga
VM3849:3 Akhabue Oluwasanya Joshua
VM3849:3 Ayodele Oluwakemi
VM3849:3 Mercy Anu
VM3849:3 Haydemola Haydeyemi Hayndrew
VM3849:3 Fowowe Holurwahfunmihlaryoh Janet
VM3849:3 Olori Oluwatosin Fadare
VM3849:3 Sandra Elom
VM3849:3 Adelakun Oluwasegun Samson
VM3849:3 Agbeyangi SK Pastor
VM3849:3 Ebere Onuzulike Christian
VM3849:3 Hardeyphillar Horluwartobiloba Hardeybowale
VM3849:3 Oluwatayo Exalted
VM3849:3 Shalom Gabriel
VM3849:3 James Okeowo
VM3849:3 David Ajala Moses
VM3849:3 Segun Oluwole
VM3849:3 Homorthosho Yohmahdeh Quasim
VM3849:3 Adeola Oluwatobbhy Ige
VM3849:3 Edebor Jeffrey
VM3849:3 Yemi Aworuwa-Olangbin
VM3849:3 Amulegbola Horlarmi
VM3849:3 Oluwafemi Olutunbi
VM3849:3 Asaolu Bolaji Grace
VM3849:3 Omololu 'Joan Fagbule
VM3849:3 Pst Olumuyiwa Adeosun
VM3849:3 Ajayi Oluwatomisin
VM3849:3 Sunday Joseph
VM3849:3 Taiwo Adeagbo T-Percussions
VM3849:3 Oluwaseun Adeosun
VM3849:3 Innocent Matthew
VM3849:3 Matthew Olufade
VM3849:3 Odeleke Tobi
VM3849:3 Ibukun Olufade Adekola
VM3849:3 Nesto Paul
VM3849:3 Ezekiel Meek Merry
VM3849:3 Daniel Adeoye
VM3849:3 Kola Jino
VM3849:3 Jude Martins Alisigwe
VM3849:3 Samson Oluwasegun Okunola
VM3849:3 Ojuko Ayo
VM3849:3 Emmanuel Showcun
VM3849:3 Oluseye Orimogunje
VM3849:3 Isaiah D Adelakun
VM3849:3 Samson Jedafe
VM3849:3 Rotimi Adegelu
VM3849:3 Daniel Oluwatobiloba
VM3849:3 Pastor Phillip Adelakun
VM3849:3 Ayobami Adeniyi
VM3849:3 Abigail Olufade
VM3849:3 Comfort Omotola
VM3849:3 Tac Nigeria
VM3849:3 Tac Oriola
VM3849:3 Ukäpsîñö Dävïdz
VM3849:3 Harrison Tom Tommbatstic
VM3849:3 Pst Sam Alao
VM3849:3 Tac Ebute Metta Assembly
VM3849:3 Ojuekanmi Ruth
VM3849:3 Abayomi Rowlands
VM3849:3 Ebere Onuzulike Christian
VM3849:3 Babs D Solochopper
VM3849:3 Adebowale Adesanya
VM3849:3 Eniawifun John
VM3849:3 Chijioke Udenna
VM3849:3 Daniel Segun Fashinu
VM3849:3 Titilayomi Aisida Awoluyi
VM3849:3 Azeez Opeyemi Rukayat
VM3849:3 Thompson Eboye
VM3849:3 Olujitan Elijah Haryourholuwah
VM3849:3 Remilekun Aiyedogbon
VM3849:3 Taiwo Tim Oyeleke
VM3849:3 Dickson Sam Edidong
VM3849:3 Deborah Adewole
VM3849:3 Odunowo Korede
VM3849:3 Hardeylarkun Haryorbammy Holuwarseun
VM3849:3 Odunayomi Esther Praise
VM3849:3 Eniawifun Moyin
VM3849:3 Yusuf Ayuba-olawale Alao
VM3849:3 Owonikoko Shakeerah
VM3849:3 Holluwharmodupe Hollayemi Adejube
VM3849:3 Akinduro Esther
VM3849:3 Elder Adeyemi Peter
VM3849:3 Tacn Powerline
VM3849:3 Rebecca Folashade Akintola
VM3849:3 Mike Temidayo MT
VM3849:3 Idris Abimbola
VM3849:3 Maradesa Grace Ebunoluwa
VM3849:3 Pretty Gift
VM3849:3 Racheal Abiodun Olubunmi
VM3849:3 Aborisade Philips O
VM3849:3 Holayemi Alao
VM3849:3 Gbenga Ipoade
VM3849:3 Hasna Mehamed
VM3849:3 Elizabeth Betsyfun
VM3849:3 Maradesa Mary
VM3849:3 Aborisade Seye
VM3849:3 Temitope Abe
VM3849:3 Jide Jelili Rafiu
VM3849:3 Kehinde Oladunjoye
VM3849:3 Esther Tosin
VM3849:3 Abigael Oludeyi-Akintola
VM3849:3 Herdeisanwo Dahniel
VM3849:3 Austin Ifechukwude Osammor
VM3849:3 Micheal Olusegun Gbadero
VM3849:3 Jemilu Olarewaju
VM3849:3 Tunmi Adegelu Ogunlade
VM3849:3 Akindele Funmilayo
VM3849:3 Olukefty Comedian
VM3849:3 Arise Lateef
VM3849:3 Bolaji Oladunjoye
VM3849:3 Ajibesin Olawale
VM3849:3 Benjamin Olusanya Gcon
VM3849:3 John Save
VM3849:3 Deborah Ososipe
VM3849:3 Bamidele Adeyemi
VM3849:3 Raheem Yusuf
VM3849:3 Mhiz Anjhorlharholuwah Peace
VM3849:3 Enoch Mary
VM3849:3 Ukaps M Paul
VM3849:3 Paul Ukaps
VM3849:3 Bolaji Grace
VM3849:3 Olufade Samson
VM3849:3 Adigun Ayo
VM3849:3 Seun Adigun
VM3849:3 Gbenga Oniyelu
VM3849:3 Esther Isaiah
VM3849:3 Opeyemi Christanah
VM3849:3 Olaoluwa Elizabeth
VM3849:3 Oladele Taiwo
VM3849:3 Oladipupo Oluwadamilola
VM3849:3 Samuel Akinola
VM3849:3 Isarel Olubamisaye
VM3849:3 Adeoye Tope Lasisi
VM3849:3 David Ukap
VM3849:3 Olajumoke Julianah
VM3849:3 Daniel Stephen
VM3849:3 Ojo Peter
VM3849:3 Saheed Moses
VM3849:3 Fasoyin Josiah
VM3849:3 Akhabue Joshua Oluwasanya
VM3849:3 Adekusibe Olaide
VM3849:3 Adegbite Adewale Abiodun
VM3849:3 Ashaolu Bolaji
VM3849:3 Edmond Safarino Kays
VM3849:3 Adeoye Abraham
VM3849:3 Joshua Olaoluwa Oludiya
VM3849:3 Osisami Oluwasholah Pelumi Megaboi
VM3849:3 Seun Aina
VM3849:3 Jaco Baba Tee
VM3849:3 Jemilu Sanni
VM3849:3 John Adeshina Aderoju
VM3849:3 Toby Wilson
VM3849:3 Ruth Harrison
VM3849:3 Samuel Adejuyitan
VM3849:3 Ese Eguaokun - Oduntan
VM3849:3 Udeh Ikechukwu Precious
VM3849:3 Funsho Taiwo
VM3849:3 Adekusibe Olaide
VM3849:3 Ajuwon Temiloluwa B
VM3849:3 Pelumi Adedapo
VM3849:3 Olugbenga Opa-Aje
VM3849:3 Oru Gideon
VM3849:3 Man Able
VM3849:3 Omolara Bukky Koyode
VM3849:3 Afolabi Tolulope
VM3849:3 Chike Ofoma
VM3849:3 Tac Lawna London Assembly
VM3849:3 Chidi Sunday
VM3849:3 Emmanuel Kunle
VM3849:3 Ameh Ibro
VM3849:3 Blessing Busayomi
VM3849:3 Lexzy Gee Sam
VM3849:3 Taiwo Olaoye
VM3849:3 Teebass Oyebode
VM3849:3 Hassan Kaz Tayo
VM3849:3 Alex Akinyele Adelakun
VM3849:3 Lawna Missions Dept
VM3849:3 Repairer Joshua Ojomo
VM3849:3 Tunde Ogunyode
VM3849:3 Sunday Iyiola
VM3849:3 Tolulope Olaleye
VM3849:3 Joel Kunleola
VM3849:3 Ariyo Kolade
VM3849:3 Adeagbo Adams
VM3849:3 Debby Oyeyinka
VM3849:3 KoredeBello Gowin Mavin
VM3849:3 Nathaniel Owoade
VM3849:3 Rajiy Khareemot Horiade
VM3849:3 Olayemi Joseph
VM3849:3 Bebeto O. Uwumwonse
VM3849:3 Iyanda Abiodun
VM3849:3 Monday S. Emmanuel
VM3849:3 Oyebode Abephe
VM3849:3 Horiemhidhu Shina
VM3849:3 Peter Chibiuzor Oruh
VM3849:3 Biodun Abolusoro-Oniyelu
VM3849:3 Oluwafunmilayo Esther
VM3849:3 Lukumon Bedemic Hafeez
VM3849:3 Ajayi Moyinoluwa Yewande
VM3849:3 Juliet Ijeoma
VM3849:3 Herkundayo Yetunde
VM3849:3 Ifeoma Precious Ofuasia
VM3849:3 Funmilayo Oladokun
VM3849:3 Malcolm Emmylee
VM3849:3 Princess Oluwanifemi Ipadeola
VM3849:3 Kiishii Rowlands
VM3849:3 Olugbade Funmilola
VM3849:3 Tolu Tobi
VM3849:3 Dcns Deborah Oniwumi
VM3849:3 Osho Bose Ipoade
VM3849:3 Anderson Adedayo Amusa
VM3849:3 Jongbo Omowunmi
VM3849:3 Claribel Seun
VM3849:3 Oluwatayo Exalted
VM3849:3 Ronke Adelaku
VM3849:3 Olufade Samson Olugbenga
VM3849:3 Oyelola Bisola
VM3849:3 Zik Oniyelu
VM3849:3 Beatrice Odunayo
VM3849:3 Ogunwumi Adeleke
VM3849:3 Damilola Phebean Adeyeye
VM3849:3 Horpeyemii Harbayomee
VM3849:3 Fatukasi Oluwatosin R
VM3849:3 Obaleye Olusegun
VM3849:3 Michael Omoniyi
VM3849:3 Prince Fadare Moses
VM3849:3 Taophick Ayantunji
VM3849:3 Chidi Udenna
VM3849:3 Dickson Esther
VM3849:3 Rebecca Oluwakemi Samuel
VM3849:3 Femi Adegelu
VM3849:3 Adankanra Olufunmi
VM3849:3 Christianah Oluwaseun Issah
VM3849:3 Victoria Omowunmi Isaiah
VM3849:3 Ayo Olufade
VM3849:3 Jacob Ayodeji Ojuekanmi
VM3849:3 Adebowale Adesanya
VM3849:3 Mrsambass SamMix
VM3849:3 Odede Dare
VM3849:3 Ajayi Moyinoluwa
VM3849:3 Gbenga Olokun
VM3849:3 Prince Hyzik
VM3849:3 Olugbenga Orimogunje
VM3849:3 Pst Solomon Falegan
VM3849:3 Edebor Racheal Osarieme
VM3849:3 Adejuyitan Folu Ogbe
VM3849:3 Augustine Umoh
VM3849:3 Hope Tolulope Orimogunje
VM3849:3 Olori Harrison Uduak
VM3849:3 Holutaryor Iyax
VM3849:3 Adegboyega Becky Oluwaseun
VM3849:3 Awopeju Emmanuel Akinyemi
VM3849:3 Faith Akindele
VM3849:3 Ibukun Kayode
VM3849:3 Oginni Olaniran Omoseeke Olabisi
VM3849:3 Abigail Oluwatayo Akogwu
VM3849:3 Lawal Damilare Hodeyemi Okanlawon
VM3849:3 Pauloo Adeleke Adegboyega
VM3849:3 Olori OluwaDamilola OluwaSeun Falegan
VM3849:3 Oluwaseyi Agekameh
VM3849:3 Oniwumi Timothy Damilola
VM3849:3 Emmanuel Abayomi Akinyemi
VM3849:3 Elder Peter Boaz Adeyemi
VM3849:3 Bolaji Shukhurhat Calmherniola
VM3849:3 Okoro Onye Daniel
VM3849:3 Olujitan Elijah
VM3849:3 Pst Uyeh
VM3849:3 Babatunde Samson Ayodeji
VM3849:3 Tac Campusministry
VM3849:3 Adeyemi Aborishade
VM3849:3 Chidima Ofuasia
VM3849:3 Lanre Orimogunje
VM3849:3 Oloyede Adelola
VM3849:3 Iyanu Elizabeth Areola
VM3849:3 Tunde Damola
VM3849:3 Olujitan Stephen
VM3849:3 Daramola Babalola
VM3849:3 Oreke Fola Unique
VM3849:3 Adeowu Oluwakemi Rebecca
VM3849:3 Rhoda Amusan Kolawole
VM3849:3 Tac Papa Tac Papa
VM3849:3 David's Vine
VM3849:3 Lekan Tunji
VM3849:3 Olalere Yusuf
VM3849:3 Apostolic Church Anniversary
VM3849:3 Oluwatobi Olusojjy Aiyedogbon
VM3849:3 Soji Lawrence
VM3849:3 Samlex Ubaka
VM3849:3 Chuks Ofuasia
VM3849:3 Eniola Ahmed
VM3849:3 Titilayo Adeyemi Esther
VM3849:3 Adenike Elizabeth Onifade
VM3849:3 Asaph Maakun Abraham Emir
VM3849:3 Abel Oluwatosin Adekola
VM3849:3 Rotr Akinwale Olujimi Olusegun
VM3849:3 Akinsohun Oluwaseun Taiwo
VM3849:3 Oyebode Moses Teeprinz
VM3849:3 Iyanu Elijah Odeleke
VM3849:3 Tee Master
VM3849:3 Johnson Baso
VM3849:3 Oyebode Oluwafemi
VM3849:3 Victoria Baso
VM3849:3 Peter Etien
VM3849:3 Akinwale Omolara Ore-oluwa
VM3849:3 Johnson Baso
VM3849:3 Olushola Kadir
VM3849:3 Tac Ilupeju Assembly
VM3849:3 Mercy Blessing Odeleke
VM3849:3 Kemi Adewuyi
VM3849:3 Adegboye Abidemi Olayinka
VM3849:3 Oluwafemi Edwards
VM3849:3 Kiloo Games
VM3849:3 City Racing 3D Community
VM3849:3 Michael Allen Comedy
VM3849:3 Criminal Case
VM3849:3 247NewsNigeria.com
VM3849:3 The Apostolic Church, Nigeria
VM3849:3 Pastor Phillip Adelakun
VM3849:3 Sahara Reporters