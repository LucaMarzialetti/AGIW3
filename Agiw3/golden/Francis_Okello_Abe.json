{
    "PATTERN": {
        "tel": [],
        "addr": [
            "SL3 8LS",
            "SL3 8LS",
            "NG13 8AP"
        ],
        "email": []
    },
    "NER": {
        "LOC": [
            "Langley",
            "UK"
        ],
        "ORG": [
            "Marish Court",
            "Company Check Ltd",
            "United Kingdom"
        ],
        "PER": [
            "MR FRANK OKELLO ABE",
            "director",
            "Secretary"
        ]
    },
    "url": "https://companycheck.co.uk/director/913339122/MR-FRANK-OKELLO-ABE/companies"
}